helix-core/src/auto_pairs.rs

```diff
@@ -321,13 +315,13 @@ fn handle_close(doc: &Rope, selection: &Selection, pair: &Pair) -> Transaction {
             // return transaction that moves past close
             (cursor, cursor, None) // no-op
         } else {
-            len_inserted += pair.close.len_utf8();
+            len_inserted = 1;
```

Comment:
Is this an intentional change from `+=` to `=`?

Reply:
Yes. It was just being added to 0 before. Doesn't really make a difference if it's added or assigned.